mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
> to get 25 fps you need to enter "pal"

Oh is it the other way around? Then you can do this.

```python
def mayafpsconverter(fps):
  return {
    "": None,
    "game": "15",
    "film": "24",
    "pal": "25",
    ...
  }.get(fps, fps + "fps")
```

Put the value you expect in the variable on the left, and the value you want it to convert to on the right.